include/linux/kernel.h

```diff
@@ -156,6 +156,7 @@
  * abs() should not be used for 64-bit types (s64, u64, long long) - use abs64()
  * for those.
  */
+#ifndef CONFIG_SEL4
```

Comment:
Whilst I don't have a counter-argument, it's not clear to me why we had to protect by adding `#ifndef CONFIG_SYS` here (and elsewhere).

Reply:
This prevents another name clash; `abs` is already provided by the Musl C library.